src/pod.rs

```diff
@@ -0,0 +1,320 @@
+use self::super::{Error, guarded_transmute_vec_permissive, guarded_transmute_many_permissive, guarded_transmute_many_pedantic, guarded_transmute_vec_pedantic,
+                  guarded_transmute_pedantic, guarded_transmute_vec, guarded_transmute_many, guarded_transmute};
+
+
+/// Type that can be non-`unsafe`ly transmuted into
+///
+/// In most cases this is a [*POD class*](http://eel.is/c++draft/class#10)
+/// or a [*trivially copyable class*](http://eel.is/c++draft/class#6).
+///
+/// Marker trait for `guarded_transmute_pod_*()` functions.
+///
+/// *Nota bene*: `bool`s aren't *actually* non-`unsafe` to transmute, because they're restricted to being `0` or `1`,
+/// which means it's UB to transmute an arbitrary byte to into a `bool`.
+///
+/// *Nota bene*: if you transmute into a floating-point type you will have a chance to create signaling NaNs
+/// (judgement's still out as to whether they're legal).
```

Comment:
Well, it's not like they're illegal, but signaling NaNs are usually undesirable here. They can be converted to quiet NaNs in separate functions.

Reply:
I'm taking "separate functions" as "user normalises" (because, (a) that's out of scope, and (b) do you know what user wants)?

Reply:
I actually meant that we should include it. It has potential use cases (do want), and if the user wants sNaNs anyway, we can invite them to use the unsafe functions.